app/api/items/ask_hint.robustness.feature

```diff
@@ -0,0 +1,154 @@
+Feature: Ask for a hint - robustness
```

Comment:
what if "askedHint" is an empty string, missing or nil? (missing or nil is maybe just handled by validation but empty string worths a test I think) I assume we consider it a hint anyway.

Reply:
How should we handle askedHint = nil?

Reply:
The simple would be to reject it, it does not really make sense IMO.

Reply:
fixed